internal/command/jsonformat/differ/value.go

```diff
@@ -72,7 +72,7 @@ type Value struct {
 	// ReplacePaths generally contains nested slices that describe paths to
 	// elements or attributes that are causing the overall resource to be
 	// replaced.
```

Comment:
A few minor post-review suggestions for this comment and struct member:

- The "generally" in the comment seems a bit of a weasel word, implying that sometimes it contains other things. I don't think it does, so we could be more certain here.
- Since we're only expecting to decode our own data, can we be more strict with the type? Maybe defining our own package-scoped equivalent of a `cty.PathSet` if necessary, and parsing the data into that when we create the `Value`.
- Failing that, explaining why this is a `[]interface{}` type would help. I think it's actually reasonable for it to be a `[][]interface{}`, as it's a set of traversals. We're using `interface{}` here because elements can be either string or int.

Reply:
I think this is a good idea, but I will do it in a separate follow up PR. Partly because it makes merging my next changes easier, but also because I'd like your feedback on it and I don't want to slow down finally closing off the chain.